### PR TITLE
Add two mentors to the ActsGpuRace proposal

### PR DESCRIPTION
**Pull request by chavid**
Implies to add a name to the mentor's list, and stanford as organisation.

### CODE REVIEW

**Comment by aflp91** [comment]
Should Stanford University be listed in the header of the proposal?

File: _gsocproposals/2019/proposal_ActsGpuRace.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ year: 2019
 organization: 
  - LAL
  - MdS
+ - CERN
```
Comment (agheata) [inline _gsocproposals/2019/proposal_ActsGpuRace.md]: since Lauren is from Stanford, add:
  - Stanford University

**Comment by JavierCVilla** [comment]
@chavid It is not your fault, I introduced a bug in #506.

I have just issued a Pr #510 , to fix it. You can revert your last commit, it should have worked without my bug.

Sorry for the inconveniences.

**Comment by JavierCVilla** [comment]
@chavid Can you rebase now?

**Comment by chavid** [comment]
I prefer the "merge" approach ;)

**Comment by chavid** [comment]
CI in progress...

File: _gsocproposals/2019/proposal_ActsGpuRace.md (hunk 1 above)
Comment (chavid) [reply _gsocproposals/2019/proposal_ActsGpuRace.md]: Yes. Done.

**Comment by chavid** [comment]
Something still to be done on my side ?

**Comment by chavid** [comment]
Thanks :)

**Pull request merged**